packages/mui-base/src/useTooltip/useTooltipOpenState.ts

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import useEventCallback from '@mui/utils/useEventCallback';
+import type { OpenChangeReason } from '@floating-ui/react';
+import type {
+  UseTooltipOpenStateParameters,
+  UseTooltipOpenStateReturnValue,
+} from './useTooltip.types';
+import { useControlled } from '../utils/useControlled';
+/**
+ *
+ * API:
+ *
+ * - [useTooltipOpenState API](https://mui.com/base-ui/api/use-tooltip-open-state/)
+ */
+export function useTooltipOpenState(
+  params: UseTooltipOpenStateParameters,
+): UseTooltipOpenStateReturnValue {
```

Comment:
This Hook becomes necessary to separate the `Tooltip.Content` `useTooltip()` Hook, which deals with positioning logic, from the `open` state. This is because the `open` state is shared between the tooltip and the anchor, but not positioning-related props.

One caveat: Floating UI React doesn't properly support the external anchor logic, as `getReferenceProps` is returned from the `useFloating` (`useTooltip`) Hook inside `Tooltip.Content`. As a result, the anchor props need to be synchronized upwards to the root, which gets passed down to the anchor element. This is the drawback of deciding to place props on `Tooltip.Content` rather than the root. 

I'm going to look into supporting the "external anchor" pattern better in the library soon.